docs/release_notes/v1.0.0-rc.2.md

```diff
@@ -0,0 +1,533 @@
+
+If you previously installed Dapr on your Kubernetes cluster using the Dapr CLI, run:
+
+*Note: Make sure you're uninstalling with your existing CLI version*
+
+```bash
+dapr uninstall --kubernetes
+```
+
+If you previously installed Dapr using __Helm 3.X__:
+
+```bash
+helm uninstall dapr -n dapr-system
+```
+
+If you installed Dapr with Helm to a namespace other than `dapr-system`, modify the uninstall command above to account for that.
+
+##### Post installation
+
+Verify the control plane pods are running and are healthy:
+
+```
+$ kubectl get pods -n dapr-system
+NAME                                    READY   STATUS    RESTARTS   AGE
+dapr-dashboard-85cc6c9f7d-hzk69         1/1     Running   0          13m
+dapr-operator-84985cb985-h2zdv          1/1     Running   0          13m
+dapr-placement-server-0                 1/1     Running   0          13m
+dapr-sentry-5974c7cb6d-9bnqd            1/1     Running   0          13m
+dapr-sidecar-injector-5b5bb9454-sfmmx   1/1     Running   0          13m
+```
+
+After Dapr 1.0.0-rc.2 has been installed, perform a rolling restart for your deployments to pick up the new version of the sidecar.
+This can be done with:
+
+```
+kubectl rollout restart deploy/<deployment-name>
+```
+
+## Breaking Changes
+
+### Dapr Runtime
+
+- **FIXED** Pubsub API will now return 403 for forbidden pubsub topics. [2323](https://github.com/dapr/dapr/issues/2323)
+
+- **MOVED** Zipkin trace configuration from Zipkin exporter into Dapr core [2521](https://github.com/dapr/dapr/issues/2521)
+
+### Dapr CLI
+
+- **MADE** shorthands across commands consistent [530](https://github.com/dapr/cli/issues/530)
+
+- **UPDATED** Default standalone Configuration to not use deprecated Zipkin exporter component. [534](https://github.com/dapr/cli/issues/534)
+
+- **ADDED** `--publish-app-id` to the publish command [536](https://github.com/dapr/cli/issues/536)
+
+- **REMOVED** unused and unsupported `--image` flag from `run` command. [537](https://github.com/dapr/cli/issues/537)
+
+- **REMOVED** `redis-host` flag from `dapr init` [540](https://github.com/dapr/cli/issues/540)
+
+### .NET SDK
+
+- **RESOLVED** Dependency injection for Actors is enabled by default for ASP.NET Core projects [171](https://github.com/dapr/dotnet-sdk/issues/171)
+
+- **RESOLVED** Dapr Actors should not use BinaryFormatter [414](https://github.com/dapr/dotnet-sdk/issues/414)
+
+- **UPDATED** Naming of HTTPExtension and related types to follow .NET conventions [421](https://github.com/dapr/dotnet-sdk/issues/421)
+
+- **FIXED** Timer invocation after Actor runtime app restarts. [435](https://github.com/dapr/dotnet-sdk/issues/435)
+
+- **REMOVED** public constructor on ActorTypeInfo - this is a breaking change but unlikely to impact users as it was not useful before [461](https://github.com/dapr/dotnet-sdk/issues/461)
+
+### Go SDK
+
+- **RESOLVED** Have consistent method/parameter names across SDKs [117](https://github.com/dapr/go-sdk/issues/117)
+
+### Java SDK
```

Comment:
Similar to dotnet sdk, it'd be good to include a link to the upgrade guide here.

https://github.com/dapr/java-sdk/issues/415

Reply:
It is in the top, this is just a simple list of issues.

Reply:
Similar to above, I'd like to keep it simple so all I'm asking for is something at the end of the list like:
Note: See https://github.com/dapr/java-sdk/issues/415 for more details.

Reply:
Done.